rock_03/lod2/obj.cfg

```diff
@@ -0,0 +1,6 @@
+objload mesh.obj
+objskin mesh ../diff ../mask
+objbumpmap mesh ../norm
+objtricollide collider
+
+mdlscale 50
```

Comment:
No newline at end of file.

Reply:
yep, I not double-checked it earlier. Fixed now